tests/console/zypper_lr.pm

```diff
@@ -11,12 +11,183 @@
 use base "consoletest";
 use strict;
 use testapi;
+use utils;
+
+sub validatelr {
+    my ($args) = @_;
+
+    my $alias           = $args->{alias} || "";
+    my $product         = $args->{product};
+    my $product_channel = $args->{product_channel} || "";
+    my $version         = get_var('VERSION');
+    my $enabled_repo    = $args->{enabled_repo} || "Yes";
+    my $uri             = $args->{uri};
+
+    # FATE#320494 (PR#11460): disable installation source after installation if we register system
+    if (   (check_var('SCC_REGISTER', 'installation') and ($uri =~ m{(cd|dvd):///}))
+        or ($product_channel =~ m{(Debuginfo|Source)}))
+    {
+        $enabled_repo = "No";
+    }
+
+    if (check_var('DISTRI', 'sle')) {
+        assert_script_run "zypper lr -d | grep \"$product$version-*$product_channel.*$product$version-*$product_channel.* *| $enabled_repo.*$uri\"";
+    }
+    elsif (check_var('DISTRI', 'opensuse')) {
+        if ($product eq "openSUSE-Leap-42.1-Source-Non-Oss") {
+            script_run "zypper lr -d | grep \"$alias.*$product *| $enabled_repo.*$uri\"";
+            record_soft_failure 'boo#988736';
```

Comment:
there is little point in record_soft_failure here. The bug is on the installation medium of 42.1 and won't be fixed there - so having this as soft_fail now only makes sure we will never have a green 42.1 update test